### ISSUE TITLE
legend_font_size doesn't impact font size in legend

### ISSUE THREAD

**Issue by jhexner** [endangeredoxen/fivecentplots#71]
hey,

seems that legend_font_size doesn't actually impact the legend font size (checked in box plot)

**Comment by endangeredoxen** [comment]
You are correct.  It looks like it only changes the legend title font size.  Will fix in the next release

**Comment by jhexner** [comment]
thanks!

**Comment by endangeredoxen** [comment]
Fixed in v0.6.0 branch as part of the next release

```
df = pd.read_csv(Path(fcp.__file__).parent / 'test_data/fake_data_bar.csv')
fcp.bar(df, x='Liquid', y='pH', tick_labels_major_x_rotation=90, legend='Measurement', legend_font_size=6, legend_title_font_size=18)
```
![image](https://github.com/user-attachments/assets/a36262ca-d61c-48bb-9775-908c9dc9a35b)
